pkgs/servers/ldap/lldap/default.nix

```diff
@@ -20,28 +20,31 @@ let
 
   commonDerivationAttrs = rec {
     pname = "lldap";
-    version = "0.5.0";
+    version = "0.5.1-alpha-6aa930";
 
     src = fetchFromGitHub {
       owner = "lldap";
       repo = "lldap";
-      rev = "v${version}";
-      hash = "sha256-2MEfwppkS9l3iHPNlkJB4tJnma0xMi0AckLv6wpzy1Y=";
+      # rev = "v${version}";
+      # hash = "";
```

Comment:
```suggestion
```

Reply:
Still hoping that we'll get a release soon ...